src/components/Demographic.jsx

```diff
@@ -64,9 +61,7 @@ export function Consent() {
 
   const navigate = useNavigate();
 
-  useEffect(() => {
-    dispatch(consentShown(dateToState(DateTime.utc())));
-  }, []);
+  useEffect(() => {}, []);
```

Comment:
@yahelnachum I see this was a bug because ConsentShown was called and this is the Demographic page.  Shouldn't we add DemographicShown and capture that timestamp in the data?

Reply:
We have a consent shown and consent completed timestamp. The consent completed timestamp would be nearly the same as the demographics shown, so if we did add a demographics shown then I would remove consent completed. I think having a demographics shown would line up with all the other timestamps we have which would be good. Also, we might want to consider explicitly saving some data showing that they consented (e.g. a simple true value in the aws data would suffice), since it would be easier than explaining that given the demographics were shown then they must have had to accept on the consent page to proceed.